components/nimbus/metrics.yaml

```diff
@@ -103,7 +160,7 @@ nimbus_events:
       branch:
         type: string
         description: The branch slug/identifier that was randomly chosen
-      enrollment_id:
+      feature_id:
```

Comment:
Should this be `enrollment_id`?

I can see either making sense.

Reply:
It's `feature_id` on desktop, so this change is just to bring things to parity there. I'm not sure enrollment_id is as used/useful as we had initially thought because we don't have such an issue with profile cloning with Glean (or at least it's easier to detect in other places)